.github/scripts/generate_release.py

```diff
@@ -1,106 +1,121 @@
 import os
+import sys
 import subprocess
 import requests
-import datetime
-import json
-from pathlib import Path
-
-REPO = "digital-preservation/droid"
-RELEASE_TAG = "develop"
-API_BASE = "https://api.github.com"
-TOKEN = os.environ["GITHUB_TOKEN"]
-HEADERS = {
-    "Authorization": f"Bearer {TOKEN}",
-    "Accept": "application/vnd.github+json"
-}
-
-def run(cmd):
-    return subprocess.check_output(cmd, text=True).strip()
-
-def get_commit_sha(branch):
-    return run(["git", "rev-parse", f"origin/{branch}"])
-
-def is_ancestor(ancestor, descendant):
-    return subprocess.call(["git", "merge-base", "--is-ancestor", ancestor, descendant]) == 0
-
-def fetch_merged_prs():
-    url = f"{API_BASE}/repos/{REPO}/pulls"
-    params = {"state": "closed", "base": "develop", "per_page": 100}
-    r = requests.get(url, headers=HEADERS, params=params)
-    r.raise_for_status()
-    return [pr for pr in r.json() if pr.get("merged_at")]
-
-def get_release():
-    url = f"{API_BASE}/repos/{REPO}/releases/tags/{RELEASE_TAG}"
-    r = requests.get(url, headers=HEADERS)
-    r.raise_for_status()
-    return r.json()
-
-def update_release_notes(release_id, body):
-    url = f"{API_BASE}/repos/{REPO}/releases/{release_id}"
-    data = {"body": body}
-    r = requests.patch(url, headers=HEADERS, json=data)
-    r.raise_for_status()
-
-def delete_assets(asset_ids):
-    for asset_id in asset_ids:
-        url = f"{API_BASE}/repos/{REPO}/releases/assets/{asset_id}"
-        r = requests.delete(url, headers=HEADERS)
-        r.raise_for_status()
-        print(f"🗑️  Deleted asset {asset_id}")
-
-def upload_asset(upload_url, file_path):
-    file_name = Path(file_path).name
-    with open(file_path, "rb") as f:
-        print(f"⬆️  Uploading {file_name}...")
-        headers = HEADERS.copy()
-        headers["Content-Type"] = "application/zip"
-        r = requests.post(f"{upload_url}?name={file_name}", headers=headers, data=f)
-        r.raise_for_status()
-
-def main():
-    # Ensure Git branches are up to date
-    run(["git", "fetch", "origin", "develop", "main"])
-    develop_sha = get_commit_sha("develop")
-    main_sha = get_commit_sha("main")
-
-    print("🔍 Fetching merged PRs...")
-    prs = fetch_merged_prs()
-    new_prs = ""
+from datetime import datetime, timezone
+import glob
+
+# === CONFIGURATION ===
+REPO = "digital-preservation/droid"  # e.g., "octocat/Hello-World"
+ASSET_FILES = sorted(
+    glob.glob("droid-binary/target/droid-binary-*-bin-win64-with-jre.zip") +
+    glob.glob("droid-binary/target/droid-binary-*-bin.zip")
+)
+
+if not ASSET_FILES:
+    print("No matching files found for upload.")
+    sys.exit(1)
+
+TOKEN = os.getenv("GITHUB_TOKEN")
+
+if not TOKEN:
+    print("GITHUB_TOKEN environment variable not set.")
+    sys.exit(1)
+
+if len(sys.argv) != 2:
+    print("Usage: python create_github_release_raw.py <release_version>")
```

Comment:
Is there a clearer error message we could prepend to this? Like "Error - Expected 2 arguments..."

Reply:
This is fairly standard if you mess up a CLI tool. You get how you were supposed to do it. I'll leave it.